Rakefile

```diff
@@ -3,9 +3,19 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require_relative 'config/application'
-require 'graphql/rake_task'
+require_relative "config/application"
+require "graphql/rake_task"
 
 Rails.application.load_tasks
 
-GraphQL::RakeTask.new(schema_name: 'LagoApiSchema')
+GraphQL::RakeTask.new(
+  schema_name: "ApiSchema",
+  idl_outfile: "api_schema.graphql",
```

Comment:
What do you thing of using something like `graphql_schemas/[api|customer_portal].[graphql|json]`?

Reply:
Good spot - Done